plugins/inputs/net_response/net_response_test.go

```diff
@@ -13,42 +13,38 @@ import (
 )
 
 func TestBadProtocol(t *testing.T) {
-	var acc testutil.Accumulator
 	// Init plugin
 	c := NetResponse{
 		Protocol: "unknownprotocol",
 		Address:  ":9999",
 	}
 	// Error
-	err := c.Gather(&acc)
+	err := c.Init()
 	require.Error(t, err)
-	require.Equal(t, "bad protocol", err.Error())
+	require.Equal(t, "config option protocol: unknown choice unknownprotocol", err.Error())
```

Comment:
You might want to use `require.ErrorContains` here.

Reply:
Oh didn't know, thanks.